docs/content/en/_index.html

```diff
@@ -430,7 +430,7 @@ <h3 class="tools__item-title">Tetragon lab</h3>
                     </a>
                 </li>
                 <li class="tools__item">
-                    <a class="tools__link" href="/docs/getting-started/">
+                    <a class="tools__link" href="https://labs.iximiuz.com/tutorials/introduction-to-tetragon" target="_blank" rel="noopener noreferrer">
                         <img src="/svgs/promo-astronomy.svg" width="64" height="64" loading="lazy" alt="telescope"/>
                         <h3 class="tools__item-title">Exploring Tetragon</h3>
                         <p class="tools__item-description">A Security Observability Tool for Kubernetes, Docker, and
```

Comment:
maybe we should change the description, it's not super clear it's a lab by itself

Reply:
For specific lab or the entire section(title)?

Reply:
I was thinking when looking at this 
<img width="237" alt="image" src="https://github.com/cilium/tetragon/assets/11256051/cb31e48b-eae5-4e5d-a102-c241da0d78ce">

it's not super clear it's a lab behind the button. Maybe just add the word "lab" somewhere :)

Reply:
I see. added it!